src/plugin/file/kml/kmlparser.js

```diff
@@ -125,6 +125,13 @@ plugin.file.kml.KMLParser = function(options) {
    */
   this.styleMap_ = {};
 
+  /**
+   * The KML styles not supported by OpenLayers map.
+   * @type {!Object<string, !Object<string>>}
```

Comment:
This type is invalid (inner `Object` has a key type but no value type).

If the value is an object with arbitrary keys, the type here should be `!Object<string, !Object>`. `<string, *>` is implied when a key/value signature isn't specified.